### DISCUSSION TITLE
Got marked as off-topic for simply trying to help?

### DISCUSSION THREAD

**Discussion by IshworTM** [community/community#138084] [Discussions] [answered]
### Select Topic Area

Question

### Body

Okay, I was on discussions a while ago, and I found a post asking about their account related stuff, I wanted to help them, which is what i did but someone marked my response as "off-topic", which is pretty stupid considering my response and the "maintainer's" response was exactly the same?
Am I not understanding the whole purpose of discussions or simply just trying to be helpful isn't allowed here?? I am genuinely confused.

First discussion response that got marked as off-topic even though that is exactly what they were asking:
https://github.com/orgs/community/discussions/128618
![image](https://github.com/user-attachments/assets/7f39d9b6-90e3-42cc-b3c3-eea7b31e5562)

And my other response:
https://github.com/orgs/community/discussions/128394
![image](https://github.com/user-attachments/assets/d52adbd8-d863-4cc5-b43c-56d2a5694d91)

And another one:
https://github.com/orgs/community/discussions/128717
![image](https://github.com/user-attachments/assets/a7a57443-cdc2-4e42-9d5a-8613c635df15)

Apologies for being a crybaby but im genuinely confused as to what I did wrong? And I hope this is the correct place to ask about discussions related queries as I couldn't find any other forums.

**Comment by LiteBrite82** [comment]
Hi @IshworTM 
It looks like you received a warning back in June; your comments may have been hidden along with the warning.
I will review the hidden comments you mentioned above; are there any others?

**Reply by LiteBrite82** [reply]
I reinstated the comments you mentioned above; thank you for your patience and contributions!!!

**Reply by IshworTM** [reply, marked as answer]
Hai, I don't reckon getting any warnings beforehand but I do remember getting an email after my responses were hidden, but my intention was to simply help people with what I know, tyvm for the answer tho :)

**Reply by LiteBrite82** [reply]
@IshworTM
Please do continue to help the community any way you can; and thank you again for all of your helpful contributions! 

The warning was for suspected coordinated or inauthentic activity like rapid questions and answers in order to earn badges. (Badges are disabled for community/discussions)

Its usually posts that have an overly simple question like: "How do I search on GitHub" followed by an immediate (within minutes) answer: "Type your query in the search bar"; that get our attention. Just look out for those situations and you should be fine.
